src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(action => {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
```

Comment:
don't forget to format code properly
```suggestion
      case 'addProperties':
        Object.assign(state, action.extraData);
        break;

      case 'removeProperties':
```